stack.go

```diff
@@ -110,6 +161,21 @@ type StackUpdateOptions struct {
 	VCSRepo     *StackVCSRepo `jsonapi:"attr,vcs-repo,omitempty"`
 }
 
+func (s stacks) UpdateConfiguration(ctx context.Context, stackID string) (*Stack, error) {
```

Comment:
Nit: while we'll never mutate `stacks`, we should define these methods as pointer receivers for consistency.